src/SFE.scss

```diff
@@ -45,6 +45,11 @@
   border-bottom: 1px solid theme-color(light);
 }
 
+.bullet-list {
```

Comment:
Pulling some of these blocks out of the accessibility policy page into here, so they can be used both in the accessibility policy page and in the edit image modal.